SourceT/codec_test.cpp

```diff
@@ -0,0 +1,12 @@
+#include <gtest/gtest.h>
+#include "devilution.h"
+
+TEST(Codec, codec_get_encoded_len) {
+	uint32_t rv = dvl::codec_get_encoded_len(50);
```

Comment:
If you don't assign to a variable, you'll get better error message (it'll have `dvl::codec_get_encoded_len(50)` instead of `rv`)